### PR TITLE
Draw location connections automatically

### PR DESCRIPTION
**Pull request by Buhallin**
See Discord channel for demo video

### CODE REVIEW

File: src/core/PlayArea.ttslua [hunk 1]
```diff
@@ -81,6 +111,267 @@ function shouldSpawnTokens(card)
...
+  local cardPos2 = card2.getPosition()
+  cardPos1.y = CONNECTION_LINE_Y
+  cardPos2.y = CONNECTION_LINE_Y
+  local pos1 = self.positionToLocal(cardPos1)
+  local pos2 = self.positionToLocal(cardPos2)
+  table.insert(lines, {
+      points = { pos1, pos2 },
+      color = CONNECTION_COLOR,
+      thickness = CONNECTION_THICKNESS,
+  });
```
Comment (Chr1Z93) [inline src/core/PlayArea.ttslua:322]: What's the `;` doing here?

**Review by Chr1Z93 — COMMENTED** [review]
Review not finished, just want to publish these comments for now (will continue tomorrow)

File: src/core/PlayArea.ttslua [hunk 2]
```diff
@@ -123,11 +414,32 @@ function getInvestigatorCount()
...
+
 -- Reset the play area's tracking of which cards have had tokens spawned.
 function resetSpawnedCards()
   spawnedLocationGUIDs = {}
 end
 
 function onScenarioChanged(scenarioName)
   currentScenario = scenarioName
+  if not showLocationLinks() then
+    broadcastToAll("Automatic location connections not available for this scenario")
```
[the remark this answers is not in the document]
Comment (Buhallin) [reply src/core/PlayArea.ttslua:439]: It defaults to white, which is what I wanted for this.  Any sort of warning color (red/orange) risks being conflated with that player.

File: src/core/PlayArea.ttslua (hunk 1 above)
Comment (Buhallin) [reply src/core/PlayArea.ttslua:322]: Muscle memory can be hard to overcome.

**Review by Chr1Z93 — APPROVED** [review]

**Pull request merged**